package.json

```diff
@@ -114,17 +115,17 @@
     "lodash": "^4.17.20",
     "lodash-webpack-plugin": "^0.11.5",
     "mocha": "^8.1.1",
-    "nodemon": "^2.0.6",
+    "nodemon": "^2.0.7",
     "nyc": "^15.1.0",
     "prettier": "^2.0.5",
     "require-all": "^3.0.0",
     "rimraf": "^3.0.2",
-    "semantic-release": "^17.0.4",
+    "semantic-release": "^17.4.2",
     "sinon": "^10.0.0",
-    "typedoc": "0.19.2",
+    "typedoc": "^0.20.36",
```

Comment:
do you know if there is a reason this was pinned at a specific version?

Reply:
> do you know if there is a reason this was pinned at a specific version?

good catch - will revert it:
https://github.com/contentful/contentful-management.js/blob/master/.dependabot/config.yml#L9